frontend/public/components/utils/units.js

```diff
@@ -44,6 +44,11 @@ const TYPES = {
     space: true,
     divisor: 1000,
   },
+  cpu: {
+    units: ['millicores', 'cores'],
```

Comment:
I'm fairly sure this must be the actual values we expect in the YAML, otherwise it will break our parsing of CPU values.